rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/utils/DiskUtils.java

```diff
@@ -111,7 +110,14 @@ public static long getDirectorySizeInMB(File dirname)
     catch (Exception e) {
       log.error(String.format("Exception while calculating the size of the folder %s with exception : %s", dirname.toString(), e.toString()));
     }
-    return NumberUtils.toLong(output.toString().split("\\s+")[0]) / 1024;
+    try {
+      return Long.parseLong(output.toString().split("\\s+")[0]) / 1024;
+    }
+    catch (NumberFormatException e)
+    {
+      log.warn(String.format("Unable to calculate size of directory %s, du output returned: %s", dirname, output.toString()));
+      return -1L;
```

Comment:
we should fix its caller `getCacheSizeMB` accordingly.

Reply:
handling